generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts

```diff
@@ -26,12 +27,12 @@ import { ActiveMenuDirective } from './layouts/navbar/active-menu.directive';
         <%=angular2AppName%>AccountModule
     ],
     declarations: [
+        <%=jhiPrefixCapitalized%>LoginModalComponent,
```

Comment:
this shouldn't be imported here it should be imported in the common module and exported from there so that it can be used in all other modules